sdk/servicebus/service-bus/src/receivers/receiverCommon.ts

```diff
@@ -174,7 +174,7 @@ export function deferMessage(
       }),
     {
       spanLinks,
-      ...toSpanOptions({ entityPath, host: context.config.host }, "client"),
+      ...toSpanOptions({ entityPath, host: context.config.host }, "receive", "client"),
```

Comment:
To mirror .NET, we need to add a "settle" operation for defer

Reply:
Settle is not a valid value for `"publish" | "receive" | "process"`, so which should we use?